### DISCUSSION TITLE
Wildcard domain detection

### DISCUSSION THREAD

**Discussion by iiiusky** [projectdiscovery/dnsx#132] [Ideas]
https://github.com/projectdiscovery/dnsx/blob/fbead47ece376bad71c480dd73a4fd3b17e037c3/internal/runner/wildcard.go#L63

Is there a better solution for wildcard domain detection? This detection usually filters out some genuine resolutions

**Comment by fuzzyami** [comment]
@iiiusky can you please provide an example for a genuine resolution that is gone?
